user/urls.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from django.urls import path
+
+from user.views import (
+    CreateUserView,
+    LoginUserView,
+    ManageUserView
+)
+
+urlpatterns = [
+    path("register/", CreateUserView.as_view(), name="create"),
+    path("login/", LoginUserView.as_view(), name="login"),
+    path("update/", ManageUserView.as_view(), name="manage"),
```

Comment:
Please review the task checklist once more: 
Create serializers and views to support the following endpoints
GET/PUT/PATCH api/user/me/ - Information about user and possibility to update information about user